### PR TITLE
add regular thank you page

### PR DESCRIPTION
**Pull request by jake-kent**

### CODE REVIEW

File: Maco/order/models.py [hunk 1]
```diff
@@ -56,7 +56,9 @@ def send_contribution_notification_email(self):
 class Line(AbstractLine):
     @property
     def basket_line(self):
-        return self.order.basket.lines.get(product_id=self.product_id)
+        if self.order.basket.lines.filter(product_id=self.product_id).exists():
```
Comment (gone) [inline Maco/order/models.py]: https://stackoverflow.com/questions/1512059/django-get-an-object-form-the-db-or-none-if-nothing-matches

File: Maco/order/views.py [hunk 2]
```diff
@@ -134,5 +138,23 @@ def get_success_url(self):
...
+
+    def get_object(self):
+        # We allow superusers to force an order thank-you page for testing
+        order = None
+        if self.request.user.is_superuser:
+            if 'sitewide_contrib_id' in self.request.GET:
+                order = SitewideContribution.objects.get(
+                    id=self.request.GET['sitewide_contrib_id'])
+
+        if not order:
```
Comment (gone) [inline Maco/order/views.py]: this if doesn't make sense

Comment (jake-kent) [reply Maco/order/views.py]: this is slightly modified code from Oscar. we can remove the first part of the code (i.e. the code before this comment) but would just as soon leave this as is

Comment (jake-kent) [reply Maco/order/views.py]: I'll try to re-engineer it a bit

**Pull request merged**